packages/wrapped-keys/src/lib/generated/litActions/common/exportPrivateKey.js

```diff
@@ -0,0 +1,2 @@
+// DO NOT EDIT THIS FILE. IT IS GENERATED ON BUILD. RUN `yarn bundle` IN THE ROOT DIRECTORY TO UPDATE THIS FILE.
+module.exports = '';
```

Comment:
These are dummy files. The lit actions build step will populate all these with the actual lit action code that is sent to the nodes, instead of the IPFS CID

In the bundle without the lit actions code this will not satisfy the if (!litActionCode) { condition and therefore the IPFS CID will be used

Reply:
Perhaps we generate these files through some manifest which will tell a script what files to generate as a pre build step. So we do not need explicit files in the folder structure created by hand. This will also allow us to git ignore these skeleton files.

Reply:
This is what I want to achieve with the double bundle process. Some script of something like that to clone the wrapped-keys nx project and generate the expanded one and to minimize the overhead of maintaining another thing

Reply:
The files have to exists before running anything, at dev time. If they are not there, the shared code will fail when trying to import those files and DX will be seriously degraded

Reply:
I see, but can't a script generate these file stubs beforehand? So the DX is not affected? Maybe I am missing something.

Reply:
Why do you want a script for that? And when do you want to run it?